config/templates/blueprint-dashboard.yml.ejs

```diff
@@ -0,0 +1,6 @@
+<%
+const plan_manager = plan.manager.name;
+%>
+
+plan:
+  Manager: <%= plan_manager %>
```

Comment:
why caps ?

Reply:
This key will get displayed on dashboard as it is. 
https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/163/files#diff-fda1a509b3b7827775e98e4fcec2a0feR11

Reply:
ok , got it. thanks .